README.md

```diff
@@ -458,8 +458,9 @@ and then activate [`lsp-haskell`](https://github.com/emacs-lsp/lsp-haskell) in y
 ```lisp
 (defun dotspacemacs/user-config ()
   "..."
+  (setq lsp-haskell-process-path-pie "hie-wrapper")
```

Comment:
Is `lsp-haskell-process-path-pie` a typo? I would expect the last part to be `hie`, not `pie`

Reply:
Oh. damn it. Just 2 lines and I still couldn't get it right on the first attempt. Fixed!

Reply:
Some study showed one-liner bug fixes were the most likely to be wrong, and have to be re-done.

So you are on the curve :)